### PR TITLE
the header correction changed with navbar list

### PR DESCRIPTION
**Pull request by nandhini-converse**

### CODE REVIEW

File: app/components/ClientComponets/Navlist.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
```
Comment (subhash-converse) [inline app/components/ClientComponets/Navlist.tsx:1]: here used useState did't mention "use client" component

File: app/components/CommonComponents/Navlist.tsx [hunk 2]
```diff
@@ -1,35 +1,132 @@
...
   const [packageButtonColor, setPackageButtonColor] = useState("black");
   const [bookingButtonColor, setBookingButtonColor] = useState("black");
+  const [navActive, setNavActive] = useState<ListDetails>();
+  const [activeList, setActiveList] = useState<ListDetails | null>(null);
+  const [subCategoryColor, setSubCategoryColor] = useState<string>("#029e9d");
+
+  const pathname = usePathname();
+  console.log("path", pathname);
+
+  // const handleListClick = (listDetails: any) => {
```
Comment (subhash-converse) [inline app/components/CommonComponents/Navlist.tsx:44]: remove commented code  or else use this code future no consider this

**Review by subhash-converse — APPROVED** [review]

**Review by narendrannj — APPROVED** [review]

File: app/components/ClientComponets/DestinationRightCard.tsx [hunk 3]
```diff
@@ -66,6 +67,25 @@ export default function TopdestinationRightCard() {
       });
   }, []);
 
+  useEffect(() => {
+    const handleResize = () => {
+      const width = window.innerWidth;
+      if (width >= 768) {
+        setSlidesPerView(1);
```
Comment (narendrannj) [inline app/components/ClientComponets/DestinationRightCard.tsx:74]: remove this if/else condition

File: app/components/CommonComponents/EntriesDropDown.tsx [hunk 4]
```diff
@@ -1,3 +1,61 @@
+// "use client";
```
Comment (narendrannj) [inline app/components/CommonComponents/EntriesDropDown.tsx:1]: remove commented code lines

File: app/destination-list/[country]/PageSection.tsx [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+  const Duration = searchParams.get("duration");
+  const price = searchParams.get("price");
+
+  const handleButtonClick = (buttonType: string) => {
+    if (buttonType === "grid") {
+      setCurrentView("grid");
+    } else if (buttonType === "bars") {
+      setCurrentView("bars");
+    }
+    console.log(`Button clicked: ${buttonType}`);
```
Comment (narendrannj) [inline app/destination-list/[country]/PageSection.tsx:80]: remove console.log

**Pull request merged**